User 엔티티 관련 에러 긴급 수정

### 📄 설명
User 엔티티 빌드 도중 에러가 발생하여 긴급하게 수정합니다.

### ✅ 작업할 내용
- [x] #67

### 🙋🏻 참고 자료
```bash
Run ./gradlew build
Downloading https://services.gradle.org/distributions/gradle-8.8-bin.zip
.............10%.............20%.............30%.............40%.............50%.............60%..............[7](https://github.com/Insideouuut/Insideouuut_backend/actions/runs/10237880390/job/28321720770#step:6:8)0%.............[8](https://github.com/Insideouuut/Insideouuut_backend/actions/runs/10237880390/job/28321720770#step:6:9)0%.............90%.............100%

Welcome to Gradle 8.8!

Here are the highlights of this release:
 - Running Gradle on Java 22
 - Configurable Gradle daemon JVM
 - Improved IDE performance for large projects

For more details see https://docs.gradle.org/8.8/release-notes.html

Starting a Gradle Daemon (subsequent builds will be faster)
warning: unknown enum constant When.MAYBE
  reason: class file for javax.annotation.meta.When not found

/home/runner/work/Insideouuut_backend/Insideouuut_backend/Inside-Out/src/main/java/com/goorm/insideout/user/domain/User.java:6[9](https://github.com/Insideouuut/Insideouuut_backend/actions/runs/10237880390/job/28321720770#step:6:10): warning: @Builder will ignore the initializing expression entirely. If you want the initializing expression to serve as default, add @Builder.Default. If it is not supposed to be settable during building, make the field final.
	private List<Meeting> runningMeetings = new ArrayList<>();
	                      ^
/home/runner/work/Insideouuut_backend/Insideouuut_backend/Inside-Out/src/main/java/com/goorm/insideout/user/domain/User.java:72: warning: @Builder will ignore the initializing expression entirely. If you want the initializing expression to serve as default, add @Builder.Default. If it is not supposed to be settable during building, make the field final.
	private List<MeetingUser> meetingUsers = new ArrayList<>();
	                          ^
/home/runner/work/Insideouuut_backend/Insideouuut_backend/Inside-Out/src/main/generated/com/goorm/insideout/user/domain/QUser.java:37: error: <identifier> expected
> Task :compileJava
    public final NumberPath<<any>> meetingUsers = createNumber("meetingUsers", <any>.class);
                           ^
/home/runner/work/Insideouuut_backend/Insideouuut_backend/Inside-Out/src/main/generated/com/goorm/insideout/user/domain/QUser.java:37: error: <identifier> expected
    public final NumberPath<<any>> meetingUsers = createNumber("meetingUsers", <any>.class);
                                ^
/home/runner/work/Insideouuut_backend/Insideouuut_backend/Inside-Out/src/main/generated/com/goorm/insideout/user/domain/QUser.java:37: error: <identifier> expected
    public final NumberPath<<any>> meetingUsers = createNumber("meetingUsers", <any>.class);
                                               ^
/home/runner/work/Insideouuut_backend/Insideouuut_backend/Inside-Out/src/main/generated/com/goorm/insideout/user/domain/QUser.java:37: error: illegal start of expression
    public final NumberPath<<any>> meetingUsers = createNumber("meetingUsers", <any>.class);
                                                                                    ^
/home/runner/work/Insideouuut_backend/Insideouuut_backend/Inside-Out/src/main/generated/com/goorm/insideout/user/domain/QUser.java:37: error: <identifier> expected
    public final NumberPath<<any>> meetingUsers = createNumber("meetingUsers", <any>.class);
                                                                                     ^
/home/runner/work/Insideouuut_backend/Insideouuut_backend/Inside-Out/src/main/generated/com/goorm/insideout/user/domain/QUser.java:37: error: ';' expected
    public final NumberPath<<any>> meetingUsers = createNumber("meetingUsers", <any>.class);
                                                                                          ^
/home/runner/work/Insideouuut_backend/Insideouuut_backend/Inside-Out/src/main/generated/com/goorm/insideout/user/domain/QUser.java:49: error: <identifier> expected
    public final NumberPath<<any>> runningMeetings = createNumber("runningMeetings", <any>.class);
                           ^
/home/runner/work/Insideouuut_backend/Insideouuut_backend/Inside-Out/src/main/generated/com/goorm/insideout/user/domain/QUser.java:49: error: <identifier> expected
    public final NumberPath<<any>> runningMeetings = createNumber("runningMeetings", <any>.class);
                                ^
/home/runner/work/Insideouuut_backend/Insideouuut_backend/Inside-Out/src/main/generated/com/goorm/insideout/user/domain/QUser.java:49: error: <identifier> expected
    public final NumberPath<<any>> runningMeetings = createNumber("runningMeetings", <any>.class);
                                                  ^
/home/runner/work/Insideouuut_backend/Insideouuut_backend/Inside-Out/src/main/generated/com/goorm/insideout/user/domain/QUser.java:49: error: illegal start of expression
    public final NumberPath<<any>> runningMeetings = createNumber("runningMeetings", <any>.class);
                                                                                          ^
/home/runner/work/Insideouuut_backend/Insideouuut_backend/Inside-Out/src/main/generated/com/goorm/insideout/user/domain/QUser.java:49: error: <identifier> expected
    public final NumberPath<<any>> runningMeetings = createNumber("runningMeetings", <any>.class);
                                                                                           ^
/home/runner/work/Insideouuut_backend/Insideouuut_backend/Inside-Out/src/main/generated/com/goorm/insideout/user/domain/QUser.java:49: error: ';' expected
    public final NumberPath<<any>> runningMeetings = createNumber("runningMeetings", <any>.class);
                                                                                                ^
[12](https://github.com/Insideouuut/Insideouuut_backend/actions/runs/10237880390/job/28321720770#step:6:13) errors
3 warnings


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileJava'.
> Compilation failed; see the compiler error output for details.

* Try:
> Run with --info option to get more log output.
> Run with --scan to get full insights.

BUILD FAILED in 34s
> Task :compileJava FAILED
1 actionable task: 1 executed
Error: Process completed with exit code 1.
0s
0s
```

### ⏰ 추정 시간
20분

### ⏰ 소요 시간
새로운 기능 추가 소요 시간에 대해 작성해주세요.

--- a/Inside-Out/src/main/java/com/goorm/insideout/user/domain/User.java
+++ b/Inside-Out/src/main/java/com/goorm/insideout/user/domain/User.java
@@ -64,12 +64,6 @@ public class User {
 
 	@Enumerated(EnumType.STRING)
 	private Gender gender;
-  
-	@OneToMany(mappedBy = "host", cascade = CascadeType.ALL)
-	private List<Meeting> runningMeetings = new ArrayList<>();
-
-	@OneToMany(mappedBy = "user", cascade = CascadeType.ALL)
-	private List<MeetingUser> meetingUsers = new ArrayList<>();
 
 	public void increaseMannerTemperature(){
 		BigDecimal temp = this.getMannerTemp().add(BigDecimal.valueOf(0.1));

